### DISCUSSION TITLE
Access to '' is not allowed

### DISCUSSION THREAD

**Discussion by DSAHakan** [SafeExamBrowser/seb-win-refactoring#489] [Q&A]
We are planning to use SEB for a course in financial statistics where the the students go to finance.yahoo.com via a link from our start-URL to get data.

In the seb-file we use URL-filtering and there allow finance.yahoo.com and consent.yahoo.com
But it is not possible to get there because the following dialog box appear:
![image](https://user-images.githubusercontent.com/115719295/195626411-e27e15f6-0855-48e0-8e06-cf10a95b6937.png)

How to get around this problem?

SEB 3.3.0.354 / Windows 10

**Comment by dbuechel** [comment]
Try to set the "Show URLs" setting to another (less restrictive) value so that you can evaluate which URL exactly is being blocked (see https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane). Then you should be able to either extend or change your existing rule set.
